docs/snippets/react/msw-addon-configure-handlers-graphql.js.mdx

```diff
@@ -63,8 +63,8 @@ export const MockedSuccess = {
         graphql.query('AllInfoQuery', () => {
           return new HttpResponse.json({
             data: {
-              allFilms: {
-                films,
+              AllInfo: {
```

Comment:
why is this PascalCase?

Reply:
To match the actual query in the component. But perhaps it would be better to update the query to use camelCase instead. I'll do that now.